lib/settings_base.py

```diff
@@ -229,6 +229,11 @@ def lazy_langs(languages):
 # Tarballs in DUMPED_USERS_PATH deleted 30 days after they have been written.
 DUMPED_USERS_DAYS_DELETE = 3600 * 24 * 30
 
+# path that isn't just one / deep, and doesn't require any locale or app.
+SUPPORTED_NONAPPS_NONLOCALES_DEEP = (
```

Comment:
I don't like the "deep" thing. I don't quite understand it. Can it be "prefix" instead?